src/Security/InteractiveSecurityHelper.php

```diff
@@ -75,4 +77,22 @@ public function guessEntryPoint(SymfonyStyle $io, array $securityData, string $a
             current($authenticators)
         );
     }
+
+    public static function guessUserClass(SymfonyStyle $io, array $securityData): string
+    {
+        if (1 === \count($securityData['security']['providers']) && isset(current($securityData['security']['providers'])['entity'])) {
+            $entityProvider = current($securityData['security']['providers']);
+            $userClass = $entityProvider['entity']['class'];
+        } else {
+            $userClass = $io->ask(
+                'Enter the User class you want to authenticate (e.g. <fg=yellow>App\\Entity\\User</>)
+ (It has to be handled by one of the firewall\'s providers)',
```

Comment:
Hmm, let's remove this. I know why you have it, but feel like it's just going to confuse people. Probably an edge case that they've successfully created a `UserInterface` User class, but have not setup a provider and need one. And even if they DO make this mistake, it's not really our job to catch here.